src/extension.ts

```diff
@@ -1,93 +1,105 @@
+} from "./commands/aksStarterWorkflow/configureStarterWorkflow";
+import {
     aksCRUDDiagnostics,
     aksBestPracticesDiagnostics,
     aksIdentitySecurityDiagnostics,
     aksNodeHealth,
     aksKnownIssuesAvailabilityPerformanceDiagnostics,
-    aksCategoryConnectivity
-} from './commands/detectors/detectors';
-import { failed } from './commands/utils/errorable';
-import aksNavToPortal from './commands/aksNavToPortal/aksNavToPortal';
-import aksClusterProperties from './commands/aksClusterProperties/aksClusterProperties';
-import aksCreateClusterNavToAzurePortal from './commands/aksCreateClusterNavToAzurePortal/aksCreateClusterNavToAzurePortal';
-import { registerAzureUtilsExtensionVariables } from '@microsoft/vscode-azext-azureutils';
-import { aksRunKubectlCommands } from './commands/aksKubectlCommands/aksKubectlCommands';
-import { longRunning } from './commands/utils/host';
-import { getClusterProperties, getKubeconfigYaml } from './commands/utils/clusters';
-import aksDeleteCluster from './commands/aksDeleteCluster/aksDeleteCluster';
-import aksRotateClusterCert from './commands/aksRotateClusterCert/aksRotateClusterCert';
-import { aksInspektorGadgetShow } from './commands/aksInspektorGadget/aksInspektorGadget';
-import aksCreateCluster from './commands/aksCreateCluster/aksCreateCluster';
-import aksReconcileCluster from './commands/aksReconcileCluster/aksReconcileCluster';
-import { aksTCPDump } from './commands/aksTCPCollection/tcpDumpCollection';
+    aksCategoryConnectivity,
+} from "./commands/detectors/detectors";
+import { failed } from "./commands/utils/errorable";
+import aksNavToPortal from "./commands/aksNavToPortal/aksNavToPortal";
+import aksClusterProperties from "./commands/aksClusterProperties/aksClusterProperties";
+import aksCreateClusterNavToAzurePortal from "./commands/aksCreateClusterNavToAzurePortal/aksCreateClusterNavToAzurePortal";
+import { registerAzureUtilsExtensionVariables } from "@microsoft/vscode-azext-azureutils";
+import { aksRunKubectlCommands } from "./commands/aksKubectlCommands/aksKubectlCommands";
+import { longRunning } from "./commands/utils/host";
+import { getClusterProperties, getKubeconfigYaml } from "./commands/utils/clusters";
+import aksDeleteCluster from "./commands/aksDeleteCluster/aksDeleteCluster";
+import aksRotateClusterCert from "./commands/aksRotateClusterCert/aksRotateClusterCert";
+import { aksInspektorGadgetShow } from "./commands/aksInspektorGadget/aksInspektorGadget";
+import aksCreateCluster from "./commands/aksCreateCluster/aksCreateCluster";
+import aksReconcileCluster from "./commands/aksReconcileCluster/aksReconcileCluster";
+import { aksTCPDump } from "./commands/aksTCPCollection/tcpDumpCollection";
 
 export async function activate(context: vscode.ExtensionContext) {
     const cloudExplorer = await k8s.extension.cloudExplorer.v1;
-    context.subscriptions.push(new Reporter(context));
+    context.subscriptions.push(new Reporter());
```

Comment:
`context` was never used here.

Reply:
Used by telemetry, verified that reporting still works correctly after the change.